src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -53,24 +56,17 @@ public class SignInManager<TUser> where TUser : class
         internal IUserClaimsPrincipalFactory<TUser> ClaimsFactory { get; private set; }
         internal IdentityOptions Options { get; private set; }
         internal ILogger Logger { get; set; }
+        protected IdentityLogger IdentityLogger { get; set; }
```

Comment:
I thought it should be accessible to the child class and hence did not make the Logger object mandatory in the constructor. Now looking back it doesn't for the child classes to create their own and I can make it internal like others